src/Common/MetricHelper.cs

```diff
@@ -500,6 +502,22 @@ private void PopulateSanitizerPropertiesFromQos(AzurePSQoSEvent qos, IDictionary
             }
         }
 
+        private void PopulateConfigInfosFromQos(AzurePSQoSEvent qos, IDictionary<string, string> eventProperties)
+        {
+
+            if (qos?.ConfigInfos != null)
+            {
+                foreach (var config in qos.ConfigInfos)
+                {
+                    eventProperties[config.ConfigKey] = config.ConfigValue;
+                    foreach (var property in config.ExtendedProperties)
+                    {
+                        eventProperties[$"{config.ConfigKey}-{property.Key}"] = property.Value;
+                    }
```

Comment:
I suggest removing this.
We are not sure what extended properties we are going to add in the future, so we'd better not put restrictions based on it.